skimage/io/sift.py

```diff
@@ -12,7 +12,15 @@
 
 
 def _sift_read(f, mode='SIFT'):
-    """Read SIFT or SURF features from a file.
+    """Read SIFT or SURF features from file.
+
+    This routine reads SIFT or SURF files generated by binary utilities from
+    http://people.cs.ubc.ca/~lowe/keypoints/ and
+    http://www.vision.ee.ethz.ch/~surf/.
+
+    This routine *does not* generate SIFT/SURF features from an image.  These
+    algorithms are patent encumbered.  Please use `skimage.feature.CENSURE`
```

Comment:
`CENSURE` is a feature detector, I guess, we should refer to `ORB` here. Alternatively, we could just refer to the feature module in general.